### PR TITLE
spreading the menu items both side of the menu button

### PR DESCRIPTION
**Pull request by OnlyRefat**
Hi, first thanks for your awesome library. I was playing with your library and forked it and made a feature.

-> spreading the menu items both side of the menu button. 

I have passed a parameter orientation="both" to specify the feature and done some coding on the source file.

Please accept the pull request if you think it can be mergeable. So that it would help others :) 

Thanks,
Rafi.

### CODE REVIEW

File: example/src/example.js [hunk 1]
```diff
@@ -37,11 +38,11 @@ export default class Example extends Component{
             border: "solid 1px #E74C3C",
             borderRadius: "40%"
           }}
-          distance={-80}
+          distance={80}
           width={50}
           height={50}
           y={500}
-          x={-150} >
+          x={150} >
```
Comment (bokuweb) [inline example/src/example.js]: Please fix x value to -150. First menu is overhanging to 4th menu.

**Comment by bokuweb** [comment]
@OnlyRefat thanks for your PR.Please fix example code.

**Comment by OnlyRefat** [comment]
The example code is now fixed. can you please check if it works. The feature is shown on the vertical direction. and others kept as same as it was.

**Comment by bokuweb** [comment]
@OnlyRefat LGTM 🎉

**Comment by OnlyRefat** [comment]
Will you please merge the master branch to gh-pages branch? so that the everyone can test the demo?

**Comment by bokuweb** [comment]
@OnlyRefat We plan to fix the example, add test and refactoring after it completes.

**Pull request merged**